SI/Sem.03/Pract.03/Prakt3Task05.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+	unsigned int number, attempts;
+	attempts = 1;
+	int newNumber;
+	cout << "Enter a number between [0,100]: "<<endl;
+	cin >> number;
+	cout << "Enter a number: "<<endl;
+	cin >> newNumber;
+	if (number >= 0 && number <= 100)
+	{
+		while (number != newNumber)
+		{ 
+			cout << "The number of attempts is: " << attempts << endl;
```

Comment:
You do not need to print the attempts each time, only after the correct number is entered.